client/flutter/lib/components/page_button.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/cupertino.dart';
 import 'package:who_app/constants.dart';
 
-class PageButton extends StatelessWidget {
+class PageButton extends StatefulWidget {
```

Comment:
Question: do we even still want the PageButton widget? It looks like it's currently only used in the onboarding flow on pages that are about to be removed/updated anyway. My suggestion would be that we do away with it and use the new Button widget wherever possible

Reply:
Okay, should I wait until it's removed from onboarding or should I remove it in this PR? Doing the latter would give a use case for the new `Button` widget but I don't want to cause problems if changes are being undergone.

Reply:
I’m using it in S-C

Reply:
Ah ok, didn't see that until tonight @advayDev1. Let's leave it in for now, we can refactor all buttons to use the new Button widget post v1. Marking this as resolved.